client/package.json

```diff
@@ -10,12 +10,14 @@
     "material-ui-icons": "^1.0.0-beta.17",
     "ramda": "^0.25.0",
     "react": "^16.2.0",
+    "react-bootstrap": "^0.32.0",
```

Comment:
Are we using react-bootstrap? if not remove line and run 'yarn' to regenerate yarn.lock